numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -4435,6 +4436,13 @@ PyArray_DescrFromType(int type)
         return NULL;
     }
     else if ((type == NPY_CHAR) || (type == NPY_CHARLTR)) {
```

Comment:
Puzzled by the distinction between `CHAR` and `CHARLTR` - and why this patch seems to translate `CHAR` into `CHARLTR` now